synapse-electron/start.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# Synapse Desktop Client - Unix Startup Script
+
+echo "Starting Synapse Desktop Client..."
+echo
+
+# Check if Node.js is installed
+if ! command -v node &> /dev/null; then
+    echo "ERROR: Node.js is not installed or not in PATH"
+    echo "Please install Node.js from https://nodejs.org/"
+    exit 1
+fi
+
+# Check if Python is installed
+if ! command -v python3 &> /dev/null; then
+    if ! command -v python &> /dev/null; then
+        echo "ERROR: Python is not installed or not in PATH"
+        echo "Please install Python from https://python.org/"
+        exit 1
+    fi
+    PYTHON_CMD="python"
+else
+    PYTHON_CMD="python3"
+fi
```

Comment:
I _think_ these usually come with the system, but looking at this, it does seem like if someone's laptop didn't have these installed, they would still have to install it correct? The good news is that I think both of these have executable installers